bangazon/urls.py

```diff
@@ -10,7 +10,7 @@
 router = routers.DefaultRouter(trailing_slash=False)
 router.register(r'products', Products, 'product')
 router.register(r'productcategories', ProductCategories, 'productcategory')
-router.register(r'lineitems', LineItems, base_name='orderproduct')
+# router.register(r'lineitems', LineItems, base_name='orderproduct')
```

Comment:
This line should not be commented out. To fix the bug from earlier remove the `basename=` from the arguments
```suggestion
router.register(r'lineitems', LineItems, 'orderproduct')
```

Reply:
Uncommented and deleted "base_name=" from line 13 in latest commit